app/src/main/java/com/compscieddy/meetinthemiddle/GroupAdapter.java

```diff
@@ -0,0 +1,53 @@
+package com.compscieddy.meetinthemiddle;
+
```

Comment:
Could you set these settings in the IDE so we have a standard spacing/indent we're all using? 
![screenshot 2016-06-07 10 49 49](https://cloud.githubusercontent.com/assets/1836653/15880083/d6ae8f98-2cde-11e6-9e99-e3eec038a8c4.png)